src/node.js

```diff
@@ -49,12 +50,30 @@ function prepareIPFSConfig () {
       s3ForcePathStyle: AWS_S3_ADDRESSING_STYLE === 'path',
       signatureVersion: AWS_S3_SIGNATURE_VERSION
     })
-    return { repo }
   } else if (IPFS_PATH) {
-    return { repo: IPFS_PATH }
+    repo = IPFS_PATH
   }
 
-  return {}
+  let swarmAddresses = [
+    '/ip4/0.0.0.0/tcp/4002',
+    '/ip4/127.0.0.1/tcp/4003/ws'
+  ]
```

Comment:
What's the purpose of adding these to the swarm?

Reply:
those are the default values according to the docs: https://github.com/ipfs/js-ipfs/blob/master/docs/config.md#swarm